scipy/special/_basic.py

```diff
@@ -2182,6 +2183,12 @@ def comb(N, k, exact=False, repetition=False):
     val : int, float, ndarray
         The total number of combinations.
 
+        .. deprecated:: 1.9.0
+            Non-integer arguments are deprecated when exact=True and comb
+            will return NaN for integral inputs in the second release after
```

Comment:
```suggestion
            will return NaN for non-integral inputs in the second release after
```

This should be "non-integral inputs" right?

Reply:
yes good spot